docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,654 @@
+
+You can also see the contents of `redis-secret` as created by Radius. Run the following command:
+
+```bash
+kubectl get secret -n demo redis-secret -o yaml
+```
+
+The output should look like the following:
+
+```txt
+>kubectl get secret -n demo redis-secret -o yaml
+apiVersion: v1
+data:
+  connectionString: cmVkaXMtcjV0Y3JyYTNkN3VoNi5kZW1vLnN2Yy5jbHVzdGVyLmxvY2FsOjYzNzksYWJvcnRDb25uZWN0PUZhbHNl
+  host: cmVkaXMtcjV0Y3JyYTNkN3VoNi5kZW1vLnN2Yy5jbHVzdGVyLmxvY2Fs
+  password: ""
+  port: NjM3OQ==
+  tls: ZmFsc2U=
+  url: cmVkaXM6Ly9yZWRpcy1yNXRjcnJhM2Q3dWg2LmRlbW8uc3ZjLmNsdXN0ZXIubG9jYWw6NjM3OS8wPw==
+  username: ""
+kind: Secret
+metadata:
+  creationTimestamp: "2023-09-13T01:49:36Z"
+  name: redis-secret
+  namespace: demo
+  ownerReferences:
+  - apiVersion: radapp.io/v1alpha3
+    blockOwnerDeletion: true
+    controller: true
+    kind: Recipe
+    name: db
+    uid: d40567a1-cd52-4984-8321-6cb8bea5f798
+  resourceVersion: "3672"
+  uid: b1613fb0-09e6-4f76-8685-02f458e173b9
+type: Opaque
+```
+
+The actual values like `connectionString` are Base64 encoded in this display. The `url` value in this secret is being used by the container to connect to the Redis Cache. For each type of Recipe, Radius stores the most-commonly used connection information for the convenience of application developers.
+
+
+## 6. Add Connection
+
+At this point we have added Radius to our existing container, and used a Recipe to create a Redis Cache. In this step you will use Radius Connections to inject settings into the container instead of explicitly managing a secret.
+
+Make sure the `app.yaml` file from `./demo/Chart/templates/app.yaml` is open in your editor.
+
+First, add another annotation. This time add the `radapp.io/connection-redis: 'db'` annotation, to `.metadata.annotations`. Order does not matter but indentation does.
+
+Here's the updated content of `metadata`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webapp
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    radapp.io/enabled: 'true'
+    radapp.io/connection-redis: 'db'
+spec:
+...
+```
+
+The `radapp.io/connection-` annotation defines a connection from the container to some other dependency. Each connection has:
+
+- A name: `redis` is the connection name this case.
+- A source: `db` is the name of the Recipe we created earlier.
+
+Connections are named because you can define many of them. The connection name is used to generate environment variables that are unique to the connection. 
+
+Since we're using a connection called `redis`, Radius will automatically define the `CONNECTION_REDIS_URL` environment variable. The prefix of `CONNECTION_REDIS_` will be combined with each of the settings that you could see in the `redis-secret` secret in the previous step.
```

Comment:
```suggestion
Since you're using a connection called `redis`, Radius will automatically define the `CONNECTION_REDIS_URL` environment variable. The prefix of `CONNECTION_REDIS_` will be combined with each of the settings that you could see in the `redis-secret` secret in the previous step.
```